Makefile

```diff
@@ -234,6 +234,10 @@ kind-create-cluster:
 kind-delete-cluster:
 	kind delete cluster --name $(KIND_NAME)
 
+kind-prepare-cluster-for-tests:
```

Comment:
I find this name a bit odd, why not just kind-load-operator-image , so it's more explicit?

Reply:
I was thinking in the future we might need to do other actions to prepare the cluster for running the tests.

Reply:
i think we better be more focused on names, just like the other targets. If we plan to do more things, then we can generalize/add new targets.

Reply:
Ok, sure.